### PR TITLE
update NuGet links

### LINKED ISSUE #116: update NuGet links to point to new repository
https://github.com/scientistproject/Scientist.net/blob/30c032cc455443e4fc187bd96a1b57663371b69d/src/Scientist/Scientist.csproj#L13

### PR DESCRIPTION
**Pull request by M-Zuber**
Fixes #116

### CODE REVIEW

**Comment by M-Zuber** [comment]
What do we want to do about these though?

https://github.com/scientistproject/Scientist.net/blob/30c032cc455443e4fc187bd96a1b57663371b69d/src/Scientist/Scientist.csproj#L10-L11

File: src/Scientist/Scientist.csproj [hunk 1]
```diff
@@ -10,8 +10,8 @@
     <Authors>GitHub</Authors>
```
Comment (shiftkey) [inline src/Scientist/Scientist.csproj]:
```suggestion
     <Authors>GitHub, Scientist.net maintainers</Authors>
```

File: src/Scientist/Scientist.csproj [hunk 2]
```diff
@@ -10,8 +10,8 @@
...
     <Description>Implementation of the Scientist library from GitHub</Description>
```
Comment (shiftkey) [inline src/Scientist/Scientist.csproj]:
```suggestion
    <Description>Implementation of the Scientist library for .net, inspired by the Ruby library</Description>
```

Comment (M-Zuber) [reply src/Scientist/Scientist.csproj]: You are going to bring The Wrath of Immo upon us with that spelling 😁 	

How does
`Implementation of the Scientist library for .NET, inspired by the Ruby library from GitHub`
sound?

Comment (shiftkey) [reply src/Scientist/Scientist.csproj]: > You are going to bring The Wrath of Immo upon us with that spelling 😁

Paging @terrajobst to this very urgent bike shed discussion :trollface:

Comment (shiftkey) [reply src/Scientist/Scientist.csproj]: > Implementation of the Scientist library for .NET, inspired by the Ruby library from GitHub

:shipit:

**Pull request merged**